embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/opentelemetry/EmbTracer.kt

```diff
@@ -20,6 +22,7 @@ internal class EmbTracer(
                 type = EmbType.Performance.Default,
                 private = false,
                 internal = false,
+                parent = Context.current().getEmbraceSpan(),
```

Comment:
Pick up the current span set on the context by default, per the expectation of the OTel API